core/main/handlers/hookedbrowsers.rb

```diff
@@ -44,6 +44,19 @@ class HookedBrowsers < BeEF::Core::Router::Router
         BeEF::Core::Logger.instance.register('Target Range', "Attempted hook from outside of permitted hooking subnet (#{request.ip}) rejected.")
         error 404
       end
+	    
+      excluded_hooking_subnet = config.get('beef.restrictions.excluded_hooking_subnet')
+      unless excluded_hooking_subnet.nil? || excluded_hooking_subnet.empty?
+        found = true
+        excluded_hooking_subnet.each do |subnet|
+          found = false if IPAddr.new(subnet).include?(request.ip)
+        end
+
+        unless found
+          BeEF::Core::Logger.instance.register('Target Range', "Attempted hook from excluded hooking subnet (#{request.ip}) rejected.")
+          error 404 
+        end
+      end
```

Comment:
The logic for `found` is inverted, which is especially confusing to read with the final `unless` conditional.

Reply:
Addressed changes recommended to @icamys in #1973. Will close this off if you're happy with that.